sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConstants.scala

```diff
@@ -40,4 +40,15 @@ private object CosmosConstants {
   object SystemProperties {
     val LineSeparator = System.getProperty("line.separator")
   }
+
+  object TableProperties {
+    val PartitionKeyDefinition = "CosmosPartitionKeyDefinition"
+    val PartitionCountSnapshot = "CosmosPartitionCountSnapshot"
+    val LastModified = "LastModifiedSnapshot"
+    val ProvisionedThroughputSnapshot = "ProvisionedThroughputSnapshot"
+    val IndexingPolicySnapshot = "IndexingPolicySnapshot"
+    val DefaultTtlInSecondsSnapshot = "DefaultTtlInSecondsSnapshot"
+    val AnalyticalStorageTtlInSecondsSnapshot = "AnalyticalStorageTtlInSecondsSnapshot"
```

Comment:
AnalyticalStorage -> AnalyticalStore

see this please:
https://docs.microsoft.com/en-us/azure/cosmos-db/analytical-store-introduction#analytical-ttl

Reply:
Fixed in next iteration